content/03.methods.md

```diff
@@ -2,13 +2,32 @@
 
 ### Datasets
 
-#### bioRxiv
-1. Describe how bioRxiv was obtained
-2. Describe metadata statistics on bioRxiv (number of preprints, number of preprints with multiple versions)
+#### BioRxiv
+BioRxiv [@doi:10.1101/833400] is a repository that contains biological and biomedical research preprints.
+We downloaded an xml snapshot of this repository on February 3, 2020 and this snapshot contained 98,023 total preprints.
+26,905 out of 98,023 are preprints that contain more than one version as bioRxiv allows users to upload revisions and corrections.
+We fixed this duplication issue by focusing only on the latest version of preprint. 
+Preprints in this snapshot can fall into one of the twenty-nine different categories with neuroscience being the most popular (Figure {@fig:biorxiv_categories}).
+Furthermore, each preprint depicts of the following content types: presenting a new result, confirming a previous finding or presenting a contradiction to a previous finding.
+Some preprints in this snapshot have even been withdrawn from bioRxiv.
+Overall, majority of preprints are writing to present new research results.
+
+![
+BioRxiv has an influx of neuroscience preprints followed by bioinformatics and then microbiology.
+This bar chart depicts the number of preprints that fall into each category of bioRxiv.
+](https://raw.githubusercontent.com/greenelab/annorxiver/35d3ea0de3c9c78e3c524736bbaada00928c88fb/biorxiv/exploratory_data_analysis/output/figures/preprint_category.png){#fig:biorxiv_categories}
+
 
 #### PubMed Central
-1. Describe how PubMed central was obtained
-2. Describe metadata statistics on PubMed central (number of articles, how many articles were processed
+PubMed Central (PMC) [@doi:10.1073/pnas.98.2.381] is a repository that contains open access to published research articles.
+We downloaded a snapshot of this repository on January 31, 2020.
+This snapshot contains 9,440 unique journals with 2,719,226 different articles (Figure {@fig:pmc_statistics}).

```

Comment:
```suggestion
This snapshot contains 2,719,226 different articles from 9,440 unique journals (Figure {@fig:pmc_statistics}).
```
Would suggest a light rephrase and this should also go in results if there's going to be a figure.